interface/wx/toplevel.h

```diff
@@ -632,17 +634,16 @@ class wxTopLevelWindow : public wxNonOwnedWindow
     virtual void ShowWithoutActivating();
 
     /**
-        Enables the maximize button to toggle full screen mode. Prior to
-        macOS 10.10 a full screen button is added to the right upper corner
-        of a window's title bar.
+        Enables the zoom button to toggle full screen mode.
 
-        Currently only available for wxOSX/Cocoa.
+        A wxFullScreenEvent is generated when the users enters or exits
+        full screen.
```

Comment:
I'm not sure if this belongs here. Maybe I just don't understand something here, but this function on its own doesn't enter/exit the full screen mode, does it? I.e. it never sends this event.

Reply:
This doesn't send the event, but it enables the button which triggers the event when the user presses the button. That's why I found it useful to include here. But maybe a addition of "... via the enter/exit full screen button" would be useful.